Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -326,10 +326,13 @@ private void RemoveCurrentProfile(MixedRealityToolkitConfigurationProfile profil
 
             Type interfaceType = typeof(T);
 
-            if (IsInitialized)
+            if (HasProfileAndIsInitialized)
```

Comment:
Looking at this again, I think I suggested the wrong thing. Let's leave this as it was and put `activeProfile != null` in the new `if`

```suggestion
            if (IsInitialized)
```